common/client.go

```diff
@@ -97,15 +167,13 @@ func (c *DatabricksClient) Authenticate() error {
 		c.fixHost()
 		return nil
 	}
-	return fmt.Errorf("authentication is not configured for provider. Please configure it\n" +
-		"through one of the following options:\n" +
-		"1. DATABRICKS_HOST + DATABRICKS_TOKEN environment variables.\n" +
-		"2. host + token provider arguments.\n" +
-		"3. azure_databricks_workspace_id + AZ CLI authentication.\n" +
-		"4. azure_databricks_workspace_id + azure_client_id + azure_client_secret + azure_tenant_id " +
-		"for Azure Service Principal authentication.\n" +
-		"5. Run `databricks configure --token` that will create ~/.databrickscfg file.\n\n" +
-		"Please check https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs#authentication for details")
+	info := ""
+	if len(c.configAttributesUsed) > 0 {
+		// TODO: add env vars
+		info = fmt.Sprintf("Attributes used: %s - %d", strings.Join(c.configAttributesUsed, ", "), len(configOptions))
```

Comment:
I would maybe changed the message to something like `%d attributes used: %s` - easier to read & understand what the number means

Reply:
I'd remove the count, it's a dummy for static check to see variable usage